straitlets/builtin_models.py

```diff
@@ -58,11 +59,27 @@ def netloc(self):
         host_port = join_filter_empty(':', self.hostname, self.port)
         return '@'.join([user_pass, host_port])
 
+    query = Dict(
+        allow_none=True,
```

Comment:
Would it be equivalent to give this a `default_value` of `{}`?  That feels slightly cleaner to me.